src/pages/Workspace.tsx

```diff
@@ -83,11 +83,11 @@ const CanvasLayer: React.FC = () => {
     const project = useWorkspaceSelector((state: WorkspaceState) => state.project.active);
     const dispatch = useWorkspaceDispatch();
 
-    const dndContext = useDndContext();
+    const [transformDisabled, setTransformDisabled] = useState(false);
 
     return (
         <TransformWrapper
-            disabled={dndContext.active !== null}
+            disabled={transformDisabled}
```

Comment:
I'm pretty sure this was for preventing transforms while moving instruments around. I know that the `mouseEnter` technically also prevents this, but I would leave it in here and just `||` the two. Reason being is in the future `dndContext` will also be used for more functionality.